gapis/gfxapi/gles/glsl_compat.go

```diff
@@ -210,6 +215,25 @@ func (t *glslTransformState) apply(child, parent interface{}) interface{} {
 				t.Type = ast.TSampler2D
 			}
 		}
+		if t.renameViewID {
```

Comment:
Seems like a good candidate for a test. Especially if we intend to switch parsers.